shared/wallets/asset/index.js

```diff
@@ -0,0 +1,163 @@
+// @flow
+import * as React from 'react'
+import * as Types from '../../constants/types/wallets'
+import {Box2, ClickableBox, Divider, Icon, Text} from '../../common-adapters'
+import {globalMargins, platformStyles, styleSheetCreate} from '../../styles'
+
+export type Props = {
```

Comment:
I started out splitting up the components for native asset and other tokens, but figured they have enough in common to make them one component.